### ISSUE TITLE
Change icon names

### ISSUE THREAD

**Issue by codeanticode** [processing/processing-android#404]
From this thread:

```
Icons names (like other android res) can only use a to z chars (lower case) an numbers from 1 to 9 + underscore: that has nothing to see with gradle, it s an android rule: if the icon is named icon-48.png that fires a fatal error.
```

Should we use a different naming convention for the icon names to copy to the sketch folder?

**Comment by GOVINDDIXIT** [comment]
Since this issue is open. I would like to work on this.

**Comment by codeanticode** [comment]
@GOVINDDIXIT thanks for your interest! Are you planning to work on this as part of a GSOC proposal?

**Comment by GOVINDDIXIT** [comment]
Yes,I am interested in GSoC 2018 and looking for some good projects like this.

**Comment by codeanticode** [comment]
Never mind... I just saw your PR, thanks a lot! I will revise it and merge if everything looks good. My only concern is that we have documented the use of the old names extensively (including in the "official" book on Processing for Android) so we may need a graceful way to transition to the new naming convention to avoid breaking people's previous sketches.

**Comment by GOVINDDIXIT** [comment]
I agree with your opinion.
